drivers/tty/serdev/pibridge_trace.h

```diff
@@ -162,42 +254,96 @@ DECLARE_EVENT_CLASS(pibridge_buffer_data_class,
 	)
 );
 
+/*
+ * pibridge_receive_io_data
+ *
+ * Info: the data received with an IO response packet.
+ * Time: after the data bytes from an IO response packet have been received.
+ */
 DEFINE_EVENT(pibridge_buffer_data_class, pibridge_receive_io_data,
 	TP_PROTO(const unsigned char *buffer, unsigned int len),
 	TP_ARGS(buffer, len)
 );
 
+/*
+ * pibridge_receive_gate_data
+ *
+ * Info: the data received with a gateway response packet.
+ * Time: after the data bytes from a gateway response packet have been
+ *        received.
+ */
 DEFINE_EVENT(pibridge_buffer_data_class, pibridge_receive_gate_data,
 	TP_PROTO(const unsigned char *buffer, unsigned int len),
 	TP_ARGS(buffer, len)
 );
 
+/*
+ * pibridge_send_io_data
+ *
+ * Info: the data sent with an IO request packet.
+ * Time: before the data portion of an IO packet is sent on the wire.
+ */
 DEFINE_EVENT(pibridge_buffer_data_class, pibridge_send_io_data,
 	TP_PROTO(const unsigned char *buffer, unsigned int len),
 	TP_ARGS(buffer, len)
 );
 
+/*
+ * pibridge_send_gate_data
+ *
+ * Info: the data sent with a gateway request packet.
+ * Time: before the data portion of a gateway packet is sent on the wire.
+ */
 DEFINE_EVENT(pibridge_buffer_data_class, pibridge_send_gate_data,
 	TP_PROTO(const unsigned char *buffer, unsigned int len),
 	TP_ARGS(buffer, len)
 );
 
+/*
+ * pibridge_send_begin
+ *
+ * Info: the bytes to send in raw format.
+ * Time: before the bytes are sent on the wire.
+ */
 DEFINE_EVENT(pibridge_buffer_data_class, pibridge_send_begin,
 	TP_PROTO(const unsigned char *buffer, unsigned int len),
 	TP_ARGS(buffer, len)
 );
 
+/*
+ * pibridge_receive_end
+ *
+ * Info: received bytes in raw format.
+ * Time: after the expected number of bytes has been received or a timeout
+ *       occurred.
+ */
 DEFINE_EVENT(pibridge_buffer_data_class, pibridge_receive_end,
 	TP_PROTO(const unsigned char *buffer, unsigned int len),
 	TP_ARGS(buffer, len)
 );
 
+/*
+ * pibridge_wakeup_receive_buffer
+ *
+ * Info: received bytes in raw format.
+ * Time: after a chunk of bytes that has been received from the serial line
+ *       that the kernel notifies about.
+ */
 DEFINE_EVENT(pibridge_buffer_data_class, pibridge_wakeup_receive_buffer,
 	TP_PROTO(const unsigned char *buffer, unsigned int len),
 	TP_ARGS(buffer, len)
 );
 
-/* Timeout */
+/*
+ * pibridge_timeout_class
+ *
+ * Print the number of received bytes within a timeout. Also print the timeout
```

Comment:
I'd reverse "received" and "bytes", i.e. "Print the number of bytes received within a timeout."